f2/src/locales/en.json

```diff
@@ -231,6 +240,8 @@
   "landingPage.title": "Report a scam or computer crime",
   "landingPage.warning": "Reporting does not always lead to an investigation and we likely won’t be able to recover what was lost or damaged.",
   "locationInfoForm.Warning": "Enter a valid Canadian postal code in the format shown or leave this blank.",
+  "locationInfoPage.postalCity": "City from postal code",
```

Comment:
in the report, the "City from postal code" and " province from postal code" both are undefined in English and French version.

Reply:
1. this is fixed under another pr
2. fixed
3. fixed
added French content for titles of each section
added French error message for self harm word

Reply:
Thank you for fixing the above issues. I found another issue:  when I click the checkbox or the radio button, the selected content is ALWAYS English. even If I choose French version to report.